cheatsheets/Authentication_Cheat_Sheet.md

```diff
@@ -23,49 +23,26 @@ For information on validating email addresses, please visit the [input validatio
 
 A key concern when using passwords for authentication is password strength. A "strong" password policy makes it difficult or even improbable for one to guess the password through either manual or automated means. The following characteristics define a strong password:
 
-### Password Length
+- Password Length
+    - **Minimum** length of the passwords should be **enforced** by the application. Passwords **shorter than 8 characters** are considered to be weak ([NIST SP800-63B](https://pages.nist.gov/800-63-3/sp800-63b.html)). 
+    - **Maximum** password length should not be set **too low**, as it will prevent users from creating passphrases. Typical maximum length is 128 characters. 
 
-Longer passwords provide a greater combination of characters and consequently make it more difficult for an attacker to guess.
+      When selecting maximum password length, limitation of hashing algorithm, that will be used for hashing passwords, should be taken into consideration because some of them [have a maximum password length](https://security.stackexchange.com/questions/39849/does-bcrypt-have-a-maximum-password-length/39851#39851).
```

Comment:
The different comma confuse the reading of the sentence (at least for 😃 ). 
I propose a little update to `When selecting maximum password length, limitation of hashing algorithm that will be used for hashing passwords, should be taken into consideration because...` (remove one comma).

Reply:
Done :)